### ISSUE TITLE
No instructions

### ISSUE THREAD

**Issue by remirr** [e-mcgee/paradox_platform#37]
There don't seem to be any instructions on how to configure this...We have just installed a new IP150+ and SP6000 panel. I installed the : homebridge-paradox-securitysystem plugin on my homebridge server running on windows. Once it is installed the JSON pops up but all that is in there is: 

{
    "platform": "Paradox",
    "name": "Paradox"
}

How do I get the Home, Sleep and Arm button to show up in Homekit - that is all I really need.

**Comment by ElphaX** [comment]
Yes, I'm also in the same boat - has anyone set this up successfully? Not much MQTT knowledge here

**Comment by deon-wentzel** [comment]
Hi,

I have IP150 (v1.34) on an MG5050 (4.96.1) which works well.

add to the config.JSON file, remember zone numbering starts at 0. 


under Platform
        {
            "platform": "Paradox",
            "ip": "ip add of IP150",
            "port": 10000,
            "password": "xxxxxxxx",
            "mqtt": false,
            "mqtturl": "ip add of mqtt:1883",
            "qos": 0,
            "retain": false,
            "mqttusername": "yyyyyyy",
            "mqttpassword": "xxxxxxxxxx",
            "logindelay": 3800,
            "polldelay": 5000,
            "waitafterlogin": 600,
            "delaybetweencmds": 250,
            "logoutdelay": 500,
            "logindelayafterreconnect": 1000,
            "accessories": [
                {
                    "name": "Alarm_Partition_1",
                    "config": {
                        "type": "Alarm",
                        "partition": 0,
                        "topic": "Paradox/partition1/"
                    }
                },
                {
                    "name": "Alarm_Partition_2",
                    "config": {
                        "type": "Alarm",
                        "partition": 1,
                        "topic": "Paradox/partition2/"
                    }
                },
                {
                    "name": "Alarm Connect",
                    "config": {
                        "type": "Connected"
                    }
                },
                {
                    "name": "Entrance Passive",
                    "config": {
                        "type": "Motion Sensor",
                        "zone": 0,
                        "topic": "Paradox/entrance/",
                        "debounceDelay": 0
                    }
                },
                {
                    "name": "Living Room Passive",
                    "config": {
                        "type": "Motion Sensor",
                        "zone": 1,
                        "topic": "Paradox/livingroom/",
                        "debounceDelay": 0
                    }
                },
                {
                    "name": "Dining Room Passive",
                    "config": {
                        "type": "Motion Sensor",
                        "zone": 2,
                        "topic": "Paradox/diningroom/",
                        "debounceDelay": 0
                    }
                },
                {
                    "name": "Kitchen Passive",
                    "config": {
                        "type": "Motion Sensor",
                        "zone": 3,
                        "topic": "Paradox/botoffice2/",
                        "debounceDelay": 0
                    }
                },
                {
                    "name": "Bedroom 1  Passive",
                    "config": {
                        "type": "Motion Sensor",
                        "zone": 4,
                        "topic": "Paradox/bedroom1/",
                        "debounceDelay": 0
                    }
                },
                {
                    "name": "Bedroom 2  Passive",
                    "config": {
                        "type": "Motion Sensor",
                        "zone": 5,
                        "topic": "Paradox/bedroom2/",
                        "debounceDelay": 0
                    }
                },
                {
                    "name": "Bedroom 3  Passive",
                    "config": {
                        "type": "Motion Sensor",
                        "zone": 6,
                        "topic": "Paradox/bedroom3/",
                        "debounceDelay": 0
                    }
                }
             }
         ]
   }

I hope this helps
